src/schema/firefox-schemas-import.js

```diff
@@ -0,0 +1,57 @@
+import fs from 'fs';
+
+import commentJson from 'comment-json';
+
+import { processSchemas } from './firefox-schemas';
+
+const SKIP_SCHEMAS = [
+  'native_host_manifest.json',
+];
+
+function readSchema(path, file) {
+  return commentJson.parse(
+    fs.readFileSync(`${path}/${file}`, 'utf-8'),
+    null, // reviver
+    true, // remove_comments
+  );
+}
+
+function writeSchema(path, file, schema) {
+  fs.writeFile(
+    `${path}/${file}`,
+    `${JSON.stringify(schema, undefined, 2)}\n`);
+}
+
+function schemaFiles(path) {
+  return fs.readdirSync(path);
+}
+
+function writeSchemasToFile(path, loadedSchemas) {
+  // Write out the schemas.
+  Object.keys(loadedSchemas).forEach((id) => {
+    const { file, schema } = loadedSchemas[id];
+    writeSchema(`${path}/../imported`, file, schema);
+  });
+}
+
+function loadSchemasFromFile(path) {
+  const schemas = [];
+  // Read the schemas into loadedSchemas.
+  schemaFiles(path).forEach((file) => {
+    if (SKIP_SCHEMAS.includes(file)) {
+      return;
+    }
+    const schema = readSchema(path, file);
+    schemas.push({ file, schema });
+  });
+  return schemas;
+}
+
+export function importSchemas() {
+  const path = process.argv[2];
+  const rawSchemas = loadSchemasFromFile(path);
+  const processedSchemas = processSchemas(rawSchemas);
+  writeSchemasToFile(path, processedSchemas);
+}
+
+importSchemas();
```

Comment:
Yeah, I'd rather it not happen at import time, that's mostly why the file management stuff got pulled out into its own file.

I've just been running this file but it probably makes more sense to have something like `bin/firefox-import` that calls this rather than having it here.